### ISSUE TITLE
Known Inhibitors of SARS-CoV-2 Nsp-13 (helicase) - Google Doc

### ISSUE THREAD

**Issue by TomkUCL** [StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B#22]
A table of known nsp13 inhibitors in the literature can be found here. Users are encouraged to add known inhibitors that are not already in the table following the format shown:

![image](https://user-images.githubusercontent.com/92033163/177123830-0b16dbca-8103-4d4b-9b8a-d4ad504f426f.png)

The google doc can be found here: 
https://docs.google.com/document/d/1aDMDrQo6alDvjMkXfTjfuPcpdEAvAt1oNj8EB6CgXtc/edit?usp=sharing

**Comment by mattodd** [comment]
OK, so we need to coalesce into one document/sheet, and we need to make sure we're collating the necessary info. DOIs for sure, and the year of publication (so we can sort if we want). Are we here identifying inhibitors of Nsp13 or are we doing viral helicases? What's our scope at this stage @ahsgc @TomkUCL ?

Ping @tmw20653 and will try to get Peter Brown on here.

**Comment by ahsgc** [comment]
@mattodd 
In this excel sheet, I added helicase inhibitors to find out any potential templates. I will narrow it down to NSP13 ASAP. 

My suggestion is to use an excel file instead of google word (for flexibility and sorting). I can merge these two documents @TomkUCL.

**Comment by mattodd** [comment]
Pasting here in case we don't already have this:
https://pubs.acs.org/doi/10.1021/acs.jpclett.0c02421
Discovery of COVID-19 Inhibitors Targeting the SARS-CoV-2 Nsp13 Helicase, Mark Andrew White*, Wei Lin, and Xiaodong Cheng* J. Phys. Chem. Lett. 2020, 11, 21,

**Comment by mattodd** [comment]
@SumeraMalik123 wrote in a recent meeting: A serial of potential drugs, including flavonoids, myricetin, scultellarein, dihydroxychromones, aryl diketoacids, 3-[(2-nitrophenyl)sulphanylmethyl]- 4prop-2-enyl-1H-1,2,4-triazole-5-thione and bismuth salts had been found by different groups to target the ATPase/ helicase activities of SARS-CoV nsp13 (Seybert et al. 2005; Yang et al. 2007a, b; Lee et al. 2009; Adedeji et al. 2012; Yu et al. 2012).
Note - not SARS2
If these refs are incorporated either in wiki or in paper, this comment can be hidden, with a note to that effect.

**Comment by ahsgc** [comment]
> 

These compounds are already listed in our common excel file with relevant citations. 

I will cross-check once again. 

@SumeraMalik123 Thank you for sharing these.

**Comment by SumeraMalik123** [comment]
Here are a few more
 licoflavone C,  a novel small compound (7-ethyl-8-mercapto-3-methyl-3,7-dihydro-1H-purine-2,6-dione also designed as EMMDPD), guanidine hydrochloride (GuHCl), ebselen, Bismuth potassium citrate, Ranitidine bismuth citrate, Bismuth citrate, three  small  organic molecules (bananin,  SSYA10-00,  and  chromone-4c), other small molecules (ChemDive ID: F306-0053 and 5645-0263),  suramin-like compounds and FPA-124. 
G-Quadruplexes (G4s) ligands may exert a reasonably antiviral activity as well

**Comment by TomkUCL** [comment]
Published October 2022:
Punicalagin as an allosteric NSP13 helicase inhibitor potently suppresses SARS-CoV-2 replication in vitro
https://www.sciencedirect.com/science/article/pii/S0166354222001589?via%3Dihub

**Comment by ahsgc** [comment]
A summary of chemotypes for SARS-CoV-2

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8188765/

[S107042802105002X.pdf](https://github.com/StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B/files/9847843/S107042802105002X.pdf)

@TomkUCL @mattodd @tmw20653

**Comment by tmw20653** [comment]
Anwar,

There is a primary sulfonamide that they list as a SARS-CoV helicase inhibitor
https://pubmed.ncbi.nlm.nih.gov/28910865/

Tim

From: Anwar Hossain ***@***.***>
Date: Sunday, October 23, 2022 at 7:55 PM
To: StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B ***@***.***>
Cc: Willson, Tim ***@***.***>, Mention ***@***.***>
Subject: Re: [StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B] Known Inhibitors of SARS-CoV-2 Nsp-13 (helicase) - Google Doc (Issue #22)

A summary of chemotypes for SARS-CoV-2

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8188765/

S107042802105002X.pdf<https://github.com/StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B/files/9847843/S107042802105002X.pdf>

@TomkUCL<https://github.com/TomkUCL> @mattodd<https://github.com/mattodd> @tmw20653<https://github.com/tmw20653>

**Comment by ahsgc** [comment]
@tmw20653 

I found several other similar molecules active against NSP13. I am now working on designing a small library (from a similarity search) to purchase or synthesize (an addition to our current thiazole series). 

From docking:

<img width="752" alt="image" src="https://user-images.githubusercontent.com/108078845/197429654-a47029bb-797b-4307-ab87-b1a91c0114b8.png">
 
Ref: https://www.sciencedirect.com/science/article/pii/S1093326322000018?via%3Dihub

Thank you, Tim!

Anwar

**Comment by tmw20653** [comment]
Anwar,

I am skeptical of Raubenolt paper, since it contains only molecular docking and no experimental data on nsp13 inhibitionn.

Tim

From: Anwar Hossain ***@***.***>
Date: Sunday, October 23, 2022 at 9:23 PM
To: StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B ***@***.***>
Cc: Willson, Tim ***@***.***>, Mention ***@***.***>
Subject: Re: [StructuralGenomicsConsortium/CNP4-Nsp13-C-terminus-B] Known Inhibitors of SARS-CoV-2 Nsp-13 (helicase) - Google Doc (Issue #22)

@tmw20653<https://github.com/tmw20653>

I found several other similar molecules active against NSP13. I am now working on designing a small library (from a similarity search) to purchase or synthesize (an addition to our current thiazole series).

From docking:

[image]<https://user-images.githubusercontent.com/108078845/197429654-a47029bb-797b-4307-ab87-b1a91c0114b8.png>

Ref: https://www.sciencedirect.com/science/article/pii/S1093326322000018?via%3Dihub

Thank you, Tim!

Anwar

**Comment by ahsgc** [comment]
Agreed! I see it is commercially available without a biological profile. I will keep track of this molecule as optional!

Anwar
